packages/building-rollup/src/createSpaConfig.js

```diff
@@ -29,24 +29,11 @@ function createSpaConfig(options) {
   );
   let outputDir = basicConfig.output.dir;
 
-  if (userOptions.rootDir) {
-    if (typeof userOptions.html === 'boolean' && userOptions.html) {
-      userOptions.html = {
-        rootDir: userOptions.rootDir,
-      };
-    }
-    if (typeof userOptions.html === 'object') {
-      userOptions.html.rootDir = userOptions.rootDir;
-    }
-  }
-
   const applySw = (htmlString, transformOptions) =>
     applyServiceWorkerRegistration(htmlString, transformOptions, userOptions, outputDir);
 
   const htmlPlugin = pluginWithOptions(html, userOptions.html, {
-    minify: !userOptions.developmentMode,
-    transform: [userOptions.injectServiceWorker && applySw].filter(isFalsy),
-    inject: false,
+    transformHtml: [userOptions.injectServiceWorker && applySw].filter(isFalsy),
```

Comment:
We will need to call html-minifier-terser here since we no longer do minification by default in the HTML plugin

Reply:
How it deals with the `rollup-plugin-terser` from here?

https://github.com/open-wc/open-wc/blob/79073a4788ce6dea6d8052f568f05867efe0063e/packages/building-rollup/src/createBasicConfig.js#L63

Reply:
That minifies the JS modules in rollup, but not the HTML